app/views/groups/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= render "shared/nav" %>
+
+<div class="transactions-container">
+  <div class="transactions-title">
+    <p class="parr"><%= @group.name %></p>
+    <p class="parr">Total: $<%= @group.expenses.sum(:amount) %></p>
+  </div>
+
+  <% @expenses.each do |expense| %>
+    <div class="transaction-card">
+      <div class="category-icon">
+        <%= image_tag @group.icon, alt: 'icon', class: "icon" %>
+      </div>
+      <div class="card-text">
+        <div class="name-amount">
+          <h4><%= expense.name %></h4>
+          <p class="amount">$<%= expense.amount %></p>
+        </div>
+        <p class="date"><%= expense.created_at.strftime('%d %B, %Y') %></p>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="fixed-button">
+    <%= link_to new_group_expense_path(group_id: @group.id) do %>
+      <button class="btn btn-success but">Add new transaction</button>
```

Comment:
- [ ] Please make sure that this button appears at the bottom of the page like in the original design. The text in the button should have a white boundary as well.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/230723045-bacb003f-5cb5-4ed4-a876-68908ed7485b.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/230722938-465bf457-f350-4340-bb2d-783a2dd08f3d.png)